github-issues.go

```diff
@@ -195,6 +199,10 @@ func populateIssues(url string) {
 			eachGitIssue.Title = elem.Title
 			eachGitIssue.Repo = elem.RepositoryURL
 			eachGitIssue.Link = elem.HTMLURL
+			eachGitIssue.CreatedAt = elem.CreatedAt.Format(layout)
+			eachGitIssue.UpdatedAt = elem.UpdatedAt.Format(layout)
+			delta := elem.UpdatedAt.Sub(elem.CreatedAt)
+			eachGitIssue.Age = (int)(delta.Hours())
```

Comment:
@harshavardhana  i believe (int) casting like that is allowed and very much go style. I don't want it to be string on purpose. I agree it could be int64.